data/campaign-dialog/67d48e4f076f4ffdb02bc83bf2290a10.cfg

```diff
@@ -0,0 +1,28 @@
+{
+	"edit_loc": [322,381],
+	"id": "67d48e4f076f4ffdb02bc83bf2290a10",
+	"image": "portraits/blink.png",
+	"image_animate_easing": "ease_in_elastic",
+	"image_animate_time": 600,
+	"image_area": [181.5,36.5,377.5,167.5],
+	"next_image_area": [41.0,27.0,214.0,143.0],
+	"options": [
+		{
+			"goto": null,
+			"text": "Continue"
+		}
+	],
+	"text": "ERROR: auto_update_window.cpp:67 : Auto-update: switched to /home/valis/argentum-age running ./bin/anura
```

Comment:
This looks like spurious text.

I'm not familiar with campaign development, but by reading the data, it seems this `text` belongs to the node identified as `67d48e4f...`, and `67d48e4f...` is mentioned in the `goto` and `related_nodes` keys of other nodes, so this suspicious text would be accessible in game.

Is that correct?